portal/templatetags/app_tags.py

```diff
@@ -84,6 +84,16 @@ def truncate(s, max_length=20):
 def is_logged_in_as_teacher(u):
     return is_logged_in(u) and u.userprofile and hasattr(u.userprofile, 'teacher')
 
+@register.filter(name='has_teacher_finished_onboarding')
+def has_teacher_finished_onboarding(u):
+    teacher = u.userprofile.teacher
+    teacher_has_class = teacher.has_school() and teacher.has_class()
+    if teacher_has_class:
+        class_ = teacher.class_()
+        return is_logged_in_as_teacher(u) and teacher_has_class and class_.has_students()
```

Comment:
Ok, so:
replace 
if...:
     return thingy
else:
    return False

with sthg like:
return is_logged_in_as_teacher(u) and class_ is not None and class_.has_students()